docs/accessibility.md

```diff
@@ -0,0 +1,89 @@
+Accessibility Guidance
+======================
+
+This document provides guidance and links to useful resources and tools for
+creating web-based content & interfaces which are accessible to the widest
+possible range of users.
+
+It is not intended to be exhaustive but rather to highlight important best
+practices and provide a shared reference which is useful when implementing or
+reviewing UI changes.
+
+## Guidelines
+
+### Mobile
+
+1. **Use media queries to adapt layout to display size** - Use `@media` queries
+   to adjust the layout depending on screen size to provide more usable
+   interfaces on small screens.
```

Comment:
One thing that could be added here - what is our approach to using media queries? For example I've seen two distinct approaches, one is to have media queries for different devices or form factors. "On iPad it should do X", "at tablet size or below it should do X", etc. Another approach I've seen is to let the boundaries of the media queries be defined by the content: "When the display size is too small to show layout X, then switch to layout Y" (media query rules involving things like pixel sizes, derived by trial and error to see at what size the layout needs to change).